build.gradle

```
@@ -34,8 +34,8 @@ allprojects {
 
     repositories {
         mavenLocal()
-        maven { url "https://dl.bintray.com/hmcts/hmcts-maven" }
         jcenter()
```

We've been adding in mavencentral over jcenter, might be good for you to do this too.

```suggestion
        mavenCentral()
        jcenter()
```